docs/source/faqs.md

```diff
@@ -0,0 +1,15 @@
+(faqs)=
+
+# Frequently Asked Questions
+
+```{list-table}
+:header-rows: 1
+:name: tab-faqs
+
+* - Question
+  - Answer
+* - Is SATRE an ISO technical standard?
+  - No. The SATRE specification aims to provide a helpful guide for TRE Operators, Developers and Builders. It can be used to evaluate existing TREs and inform how they could be improved, or inform the development process of new TREs.
+* - Does SATRE provide everything I need to build a TRE?
+  - No. The SATRE specification defines a set of stakeholder {ref}`roles <satre_roles>` and feature {ref}`capabilities <satre_pillars>` for TREs, which were decided according to these {ref}`architectural principles <satre_principles>`. It does not dictate or even suggest specific technologies that could or should be used to build a TRE.
+```
```

Comment:
* - How do I build and run a SATRE compliant TRE?
  - We encourage all TREs to publically evaluate themselves against SATRE, and we will link to these evaluations and TREs. These are a good starting point for deciding how to build/deploy your TRE- in many cases these TREs are open-source, and can be deployed for testing.